cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -500,13 +617,155 @@ After .NET Core 2.0 it is possible to automatically generate and verify the anti
 
 And then add the `[AutoValidateAntiforgeryToken]` attribute to the action result.
 
-## A9 Using components with known vulnerabilities
+## A7 Cross-Site Scripting (XSS)
```

Comment:
It miss the info link to the [XSS CS](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md) like for the LDAP injection.